invertedai/large/drive.py

```diff
@@ -0,0 +1,301 @@
+from typing import Tuple, Optional, List
+from pydantic import BaseModel, validate_call
+from math import ceil
+import asyncio
+
+import invertedai as iai
+from invertedai.large.common import Region
+from invertedai.common import Point, AgentState, AgentAttributes, RecurrentState, TrafficLightStatesDict, LightRecurrentStates
+from invertedai.api.drive import DriveResponse
+
+BUFFER_FOV = 35
+
+class Particle(BaseModel):
```

Comment:
Are we sure we want to keep using the word `Particle` here ? TreeNode or Node make more sense to me here. Or we can simply just call it Agent in this local namespace.

Reply:
This was a carry over from the old area drive, however particle is the most commonly used term for "things that are placed within a quad tree". However Agent would indeed be more effective and descriptive.